packages/dcos-integration-test/extra/test_adminrouter_open.py

```diff
@@ -31,9 +33,9 @@ class TestRedirectSecurity:
     )
     def test_redirect_host(
         self,
-        dcos_api_session,
-        path,
-        expected,
+        dcos_api_session: Any,
```

Comment:
Can we make all `dcos_api_session` or similar into `dcos_test_utils.dcos_api.DcosApiSession`?